### PR TITLE
feat: Ae coin details

### LINKED ISSUE #749: AE coin details
**Acceptance criteria**:
- A new page is visible displaying AE coin information (/tokens/AE)
- A panel is visible with the following data:
  - Symbol: AE (+ logo)
  - Price: AE ($) with also percentage change (same as dashboard)
  - Total Supply
  - Circulating Supply
  - Decimals
- Each row of the panel has tooltips

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #749 

## Demo

https://github.com/aeternity/aescan/assets/15363559/85a209a8-387e-4055-a69b-8ed9f130b625

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)

### CODE REVIEW

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
I think it's worth to check if text is placed in green filed when copy it: 
https://github.com/aeternity/aescan/assets/69896204/9f2a4278-1c3c-4adc-b270-097bed6d0ca1

Also for short names of tokens (the same on prod): 
https://github.com/aeternity/aescan/assets/69896204/92198165-f7fe-4bc5-9246-7725fa223800

**Comment by janmichek** [comment]
> I think it's worth to check if text is placed in green filed when copy it:

Fixed


> Also for short names of tokens (the same on prod) and aligning is different from prod:

The layout is good. That's the thing we changed lately

File: src/components/AeCoinPanel.vue [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+const { totalTokenSupply } = storeToRefs(useBlockchainStatsStore())
+const { fetchTotalStats } = useBlockchainStatsStore()
+
+await useAsyncData(() => Promise.all([
+  fetchTotalStats(),
+  fetchMarketStats(),
+]))
+
+const priceChangeSign = computed(() => priceChange.value > 0 ? '+' : '')
```
Comment (michele-franchi) [inline src/components/AeCoinPanel.vue:95]:
```suggestion
const priceChangeSign = computed(() => priceChange.value > 0 ? '+' : '')
```

Shouldn't it be:
```
> 0: '+'
=0: ''
<0: '-'
```

File: src/utils/hints/aeCoinHints.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const aeCoinHints = {
+  aeCoin: 'æternity native coin.',
+  tokenSymbol: 'Reserved token name.',
```
Comment (michele-franchi) [inline src/utils/hints/aeCoinHints.js:3]:
```suggestion
  tokenSymbol: 'Reserved coin name.',
```

File: src/utils/hints/aeCoinHints.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  price: 'Price of the token in AE and USD.',
```
Comment (michele-franchi) [inline src/utils/hints/aeCoinHints.js:4]:
```suggestion
  price: 'Price of the coin in AE and USD.',
```

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I suggested some minor changes.

File: src/components/AeCoinPanel.vue (hunk 1 above)
Comment (janmichek) [reply src/components/AeCoinPanel.vue]: right, fixed

**Pull request merged**